spec/controllers/abstract_jobs_controller_spec.rb

```diff
@@ -1,25 +1,22 @@
 # The majority of The Supplejack Manager code is Crown copyright (C) 2014, New Zealand Government,
-# and is licensed under the GNU General Public License, version 3. Some components are 
-# third party components that are licensed under the MIT license or otherwise publicly available. 
-# See https://github.com/DigitalNZ/supplejack_manager for details. 
-# 
-# Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs. 
+# and is licensed under the GNU General Public License, version 3. Some components are
+# third party components that are licensed under the MIT license or otherwise publicly available.
+# See https://github.com/DigitalNZ/supplejack_manager for details.
+#
+# Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs.
 # http://digitalnz.org/supplejack
 
-require "spec_helper"
+require 'spec_helper'
 
 describe AbstractJobsController do
-
-  let(:job) { mock_model(AbstractJob).as_null_object }
-
   before(:each) do
-    allow(controller).to receive(:authenticate_user!) { true }
+    sign_in create(:user)
   end
 
-  describe "GET index" do
-    it "returns active abstract jobs" do
-      expect(AbstractJob).to receive(:search).with(hash_including("status" => "active"))
-      get :index, status: "active", environment: "staging"
+  describe 'GET index' do
+    it 'returns active abstract jobs' do
+      expect(AbstractJob).to receive(:search).with(hash_including('status' => 'active'))
```

Comment:
maybe the `=>` can be changed to `:`

Reply:
Oh I'm pretty sure you can't when the keys are strings.

Reply:
Thanks bro. good to know